src/keys.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) IBM Corp. 2013,2017. All Rights Reserved.
+// Node module: loopback4-example-log-extension
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+/**
+ * Binding keys used by LogOp Component
+ */
+
+export namespace LogBindings {
+  export const METADATA = 'log.metadata';
+  export const LOG_ACTION = 'log.action.logger';
+  export const LOG_LEVEL = 'log.level';
+}
+
+export namespace LogLevel {
+  export const DEBUG = 0;
```

Comment:
Can we use `enum` type please? See https://www.typescriptlang.org/docs/handbook/enums.html#numeric-enums